### ISSUE TITLE
Installation fails on Ubuntu 18.04 - neo4j : Depends: cypher-shell (< 1.2.0)

### ISSUE THREAD

**Issue by petriusus** [neo4j/neo4j#12358]
Neo4j version: 3.5.11
Operating system: Ubuntu 18.04.3 LTS (GNU/Linux 4.15.0-72-generic x86_64)
Fresh OS installation (same happens with a VM that installed Neo4j normal a few weeks ago).

I just cannot get around this problem.

### Expected behavior
Normal installation

### Actual behavior
**First:**
```
wget -O - https://debian.neo4j.org/neotechnology.gpg.key | sudo apt-key add -
echo 'deb https://debian.neo4j.org/repo stable/' | sudo tee /etc/apt/sources.list.d/neo4j.list
sudo apt-get update
```
_All good soo far_

**But then**
```
sudo apt-get install -y neo4j=1:3.5.11
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
neo4j : Depends: cypher-shell (< 1.2.0) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

```

**Comment by petriusus** [comment]
I was able to install version 3.5.5, but not higher.

Did Neo4j forget to release updated version of cypher-shell in their repo? [Seems like it is not the first time](https://stackoverflow.com/questions/43745678/cypher-shell-not-installable-ubuntu-16-04).

**Comment by jennyowen** [comment]
Hi @petriusus  this has been my pain for the past few days :disappointed: 
This is because we released cypher-shell version 4.0.0 last week and `apt` just doesn't seem to be able to cope with the idea that some packages might have a dependency on _not_ the latest version of another package.
Doing:
```
apt list -a cypher-shell
Listing... Done
cypher-shell/stable 4.0.0 all [upgradable from: 1.1.12]
cypher-shell/stable,now 1.1.12 all [installed,upgradable to: 4.0.0]
cypher-shell/stable 1.1.11 all
cypher-shell/stable 1.1.10 all
...
```
you can see that the necessary cypher-shell packages are available and known to apt, it just chooses to error instead??

**As a workaround you can pre-install cypher-shell version 1.1.12:**
`sudo apt install -y cypher-shell=1.1.12`

**Or you can inline it when you install neo4j:**
`sudo apt install -y neo4j=1:3.5.13 cypher-shell=1.1.12`

Anyway, I may need to completely restructure the neo4j apt repository, or SOMETHING. I'm not sure there's a clean way of fixing this, and it's no doubt broken for a LOT of people.

Suggestions for how to fix this are very welcome.

**Comment by petriusus** [comment]
Love that inliner, thank you!

**Comment by petriusus** [comment]
> 
> 
> Hi @petriusus this has been my pain for the past few days 😞
> This is because we released cypher-shell version 4.0.0 last week and `apt` just doesn't seem to be able to cope with the idea that some packages might have a dependency on _not_ the latest version of another package.
> Doing:
> 
> ```
> apt list -a cypher-shell
> Listing... Done
> cypher-shell/stable 4.0.0 all [upgradable from: 1.1.12]
> cypher-shell/stable,now 1.1.12 all [installed,upgradable to: 4.0.0]
> cypher-shell/stable 1.1.11 all
> cypher-shell/stable 1.1.10 all
> ...
> ```
> 
> you can see that the necessary cypher-shell packages are available and known to apt, it just chooses to error instead??
> 
> **As a workaround you can pre-install cypher-shell version 1.1.12:**
> `sudo apt install -y cypher-shell=1.1.12`
> 
> **Or you can inline it when you install neo4j:**
> `sudo apt install -y neo4j=1:3.5.13 cypher-shell=1.1.12`
> 
> Anyway, I may need to completely restructure the neo4j apt repository, or SOMETHING. I'm not sure there's a clean way of fixing this, and it's no doubt broken for a LOT of people.
> 
> Suggestions for how to fix this are very welcome.

Love that inliner, thank you!

**Comment by jennyowen** [comment]
I've fixed this by removing the cypher-shell 4.0 deb from the repository for now. 
A more permanent solution coming soon...

**Comment by jennyowen** [comment]
I've now fixed this by completely changing the structure of our debian repository. This also meant I had to migrate the repository to debian.neo4j.**com** instead of .org so that I didn't break everything for users still on the old repository.

To be able to access 4.0 linux packages you need to use debian.neo4j.com or yum.neo4j.com. There is no way I can publish those new packages into the old repository without breaking apt again.

For new debian installation instructions, see https://neo4j.com/docs/operations-manual/current/installation/linux/debian/
